### PR TITLE
Text-button/Dropdown

### PR DESCRIPTION
**Pull request by phoebe-qian**
## Descriptions 
Allow user to create a new meeting by selecting meeting details (location, meeting type etc). Drop-down branch consists of code for the text-button AND drop-down because the dropdown functionality required the code for the text-button. 

Components are under the frontend/app/components/atoms folder. We have created sassy css files within the frontend/styles folder for both the textbox and drop-down to match the figma design. 

## Testing

Testing was done by adding these components onto the meetings/page.tsx page. 

Example of frontend components being used: 

Text Button Bare:
image

Text Button when clicked:
image

Choosing drop downs: 
image

### CODE REVIEW

File: frontend/app/components/atoms/textbutton/index.tsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import React, { useState } from "react";
+import styles from "../../../../styles/TextButton.module.scss";
+import Dropdown from "../drop-down/index";
+
+interface ButtonProps {
+  label: string;              // Text to display inside the button
+  onClick: () => void;        // Function to call when button is clicked
```
Comment (owenchen04) [inline frontend/app/components/atoms/textbutton/index.tsx:7]: when the button is pressed, we just want to call this `onClick` function and nothing else

File: frontend/app/meetings/page.tsx
Comment (owenchen04) [inline frontend/app/meetings/page.tsx:1]: please undo the changes in meetings/page.tsx

File: frontend/app/components/atoms/drop-down/index.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
+import React, { useState } from "react";
+import styles from "../../../../styles/Dropdown.module.scss";
+
+interface DropdownProps {
+  isVisible: boolean;
+}
+
+const locations = [
```
Comment (owenchen04) [inline frontend/app/components/atoms/drop-down/index.tsx:8]: these consts (locations, meetings, accounts) should be passed in as props to the dropdown component

**Review by owenchen04 — CHANGES_REQUESTED** [review]
Hi Phoebe and Alisha! Nice work so far on this ticket! Here are some high-level things to address:

- Please remove any of the `.json` or `yarn.lock` files from this commit
- Undo changes in `meetings/page.tsx`
- Remember that we are working with atomic design, so the textbutton and dropdown selector logic should not be connected at all at this point. In other words, the text button should be its own thing and should be able to be reused for any purpose (not just to trigger the dropdowns to be visible). While ultimately we'd like to put these components together into one "create meeting" organism, we're not quite there yet. When you click the text button, just have it call the onClick, which will eventually be passed in as a prop by whoever is using the button in a larger component. Similarly, the dropdown should be its own thing and shouldn't be connected to a text button at this point, nor should it have any references to the location, meetings, accounts, etc. Instead, try to create just a generic dropdown where the caller of the component can pass in their own options as a prop. This will allow us to reuse this single component for the location, Zoom acc, and group selectors, as opposed to hardcoding them within the component itself.

**Comment by phoebe-qian** [comment]
All changes and suggestions should be reflected in the new pull request: Text-dropdown 2

**Comment by phoebe-qian** [comment]
All changes and suggestions should be reflected in the new pull request: Text-dropdown 2

**Pull request closed without merging**